src/Workspaces/CSharp/Portable/Classification/SyntaxClassification/NameSyntaxClassifier.cs

```diff
@@ -105,15 +105,33 @@ protected override bool IsParentAnAttribute(SyntaxNode node)
         {
             // If everything classifies the same way, then just pick that classification.
             using var _ = PooledHashSet<ClassifiedSpan>.GetInstance(out var set);
+            using var _s = PooledHashSet<ClassifiedSpan>.GetInstance(out var sset);
+            var count = result.Count;
             foreach (var symbol in symbolInfo.CandidateSymbols)
             {
                 if (TryClassifySymbol(name, symbol, semanticModel, cancellationToken, out var classifiedSpan))
                 {
-                    set.Add(classifiedSpan);
+                    if (classifiedSpan.ClassificationType != ClassificationTypeNames.Keyword)
+                    {
+                        TryClassifyStaticSymbol(symbol, classifiedSpan.TextSpan, result);
```

Comment:
this would need comments explaining why this is being done.  it's not something that can be trivially understood just by examining the code.